openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx

```diff
@@ -695,7 +688,6 @@ const AddAlertPage = () => {
                                 </div>
                               </div>
                             ))}
-                            <Form.ErrorList errors={errors} />
```

Comment:
Why we removed errors?

Reply:
@chirag-madlani  since we are not handling any error for filters, that's why it's not required here!